src/main/java/at/hannibal2/skyhanni/features/commands/TransferCooldown.kt

```diff
@@ -0,0 +1,49 @@
+package at.hannibal2.skyhanni.features.commands
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.utils.ChatUtils
+import net.minecraft.client.Minecraft
+import net.minecraft.util.EnumChatFormatting
+import net.minecraftforge.event.world.WorldEvent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import java.util.concurrent.Executors
+
+object TransferCooldown {
+
+    private val executor = Executors.newSingleThreadExecutor() // needed because in some instances the "WorldEvent.Load" gets called multiple times
+
+    @Volatile var isTaskScheduled = false
+        private set
+
+    private val transferCooldownEndedEvent = mutableListOf<() -> Unit>()
```

Comment:
instead of a list of runnables, just save the last command and then check a given time after the word switch